docs/_basic/responding_actions.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Responding to actions
+slug: action-respond
+order: 8
+---
+
+<div class="section-content">
+There are two main ways to respond to actions. The first way (and the most common way) is using the `say` function. The `say` function sends a message back to the conversation where the incoming event took place.
+
+The second way to respond to actions is using `respond()`. This is a simple utility to use the `response_url` associated with an action.
```

Comment:
its very confusing that this is explained here and then there's a secondary section below that also explains it.